config/webpacker.yml

```diff
@@ -0,0 +1,65 @@
+# File written by Decidim generator
+# Note: You must restart bin/webpack-dev-server for changes to take effect
+
+default: &default
+  source_path: app/packs
+  source_entry_path: entrypoints
+  public_output_path: decidim-packs
+  public_root_path: public
+  webpack_compile_output: true
+  cache_path: tmp/webpacker-cache
+  extract_css: true
+
+  additional_paths:
+    - node_modules
+    - app/packs
+
+  # Reload manifest.json on all requests so we reload latest compiled packs
+  cache_manifest: false
+
+development:
+  <<: *default
+  compile: true
+  # Compile test packs to decidim decidim-packs folder
+
+  # Reference: https://webpack.js.org/configuration/dev-server/
+  dev_server:
+    https: false
+    host: localhost
+    # Notice that we use a different port (to prevent blocking the default one) as
+    # there will be at least two webpack servers running
+    port: 3035
+    # Hot Module Replacement updates modules while the application is running without a full reload
+    hmr: false
+    client:
+      # Should we show a full-screen overlay in the browser when there are compiler errors or warnings?
+      overlay: true
+      # May also be a string
+      # webSocketURL:
+      #  hostname: "0.0.0.0"
+      #  pathname: "/ws"
+      #  port: 8080
+    # Should we use gzip compression?
+    compress: true
+    # Note that apps that do not check the host are vulnerable to DNS rebinding attacks
+    allowed_hosts: "all"
+    pretty: true
+    headers:
+      'Access-Control-Allow-Origin': '*'
+    static:
+      watch:
+        ignored: '**/node_modules/**'
+
+test:
+  <<: *default
+  compile: true
+  public_output_path: packs-test
+
+production:
+  <<: *default
+
+  # Production depends on precompilation of packs prior to booting for performance.
+  compile: false
+
+  # Cache manifest.json for performance
+  cache_manifest: true
```

Comment:
Remember add the preproduction environment here and in `config/webpack/preprod.js`

Reply:
I think we're deploying in PRE using the `production` environment, at least for Rails. In that case this is not needed, amb I right?